src/locale/locales/de/messages.po

```diff
@@ -467,34 +456,33 @@ msgstr "Suchanfrage löschen"
 
 #: src/view/com/auth/login/PasswordUpdatedForm.tsx:38
 msgid "Close alert"
-msgstr "Alarm schließen"
+msgstr ""
 
 #: src/view/com/util/BottomSheetCustomBackdrop.tsx:33
 msgid "Close bottom drawer"
-msgstr "Untere Schublade schließen"
+msgstr ""
 
 #: src/view/com/lightbox/ImageViewing/components/ImageDefaultHeader.tsx:26
 msgid "Close image"
 msgstr "Bild schließen"
 
 #: src/view/com/lightbox/Lightbox.web.tsx:112
 msgid "Close image viewer"
-msgstr "Bildanzeige schließen"
+msgstr ""
```

Comment:
Was good before

Reply:
`"Bild schließen"` would also be a common alternative.